main/main.go

```diff
@@ -9,7 +12,16 @@ const (
 )
 
 func main() {
-	s := master.NewServer(bufferSize, httpPort, grpcPort)
+	config := master.ServerConfig{
+		ConnBufferSize: bufferSize,
+		GrpcPort:       grpcPort,
+		HttpPort:       httpPort,
+	}
+
+	flag.StringVar(&config.StateFileName, "state_from_file", "", "specify the state name you want to load")
+	flag.Parse()
```

Comment:
It would be great to have a bool flag like `load_latest_state`, so you don't have to specify the exact file-name if you just want the server to continue where it left off.